kafka_producer.js

```diff
@@ -58,12 +58,14 @@ producer.on('error', function (err) {
 });
 
 //Listening to Port
-oidc.on('ready', () => {
-    app.listen(5001, function () {
-        console.log('Kafka producer running at 5001');
-    });
+// oidc.on('ready', () => {
+//     app.listen(5001, function () {
+//         console.log('Kafka producer running at 5001');
+//     });
+// });
```

Comment:
Revert this, and use the config param as mentioned above.

Reply:
Added a config to enable or disable OIDC